psi4/src/read_options.cc

```diff
@@ -1586,7 +1586,9 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
             - also : lowest-energy triplets and singlets included
             - only : Only triplet states computed
              -*/
-        options.add_str("TDSCF_TRIPLETS", "none");
```

Comment:
Why the change?

Reply:
I don't the the string is ever interpreted. In `tdscf_excitations`, the `triplet` variable is passed to `TDRSCCFEngine` as a bool. I like the idea of the none/some/all specifications but I couldn't tell if that was implemented.

Reply:
You're right. This is not properly in place yet.